plaso/dependencies.py

```diff
@@ -142,12 +142,6 @@ def _CheckPythonModule(
       map(int, _VERSION_SPLIT_REGEX.split(minimum_version)))
 
   if module_version_map < minimum_version_map:
-    if not is_required:
```

Comment:
Please correct this, these changes were not implemented in l2tdevtools.

Reply:
Reverted